ansible/roles/eos/templates/t0-leaf-lag-2.j2

```diff
@@ -0,0 +1,135 @@
+{% set host = configuration[hostname] %}
```

Comment:
Content of this new file is identical with t0-leaf.j2. What is the purpose of this file? Can we just reuse the t0-leaf.j2 file?

Reply:
lag min-link is 1 and 2 respectively. Though I am not sure if this would cause any trouble. We might config this according to the topology template again?

Reply:
Ah, I missed this min-link change before creating the PR.

Reply:
Fixed now.